src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+
+  const PERIOD_WITHOUT_DISCOUNT = 2;
+  const PERIOD_WITH_SMALL_DISCOUNT = 7;
+
+  let price = days * RENT_COST;
+
+  if (days <= PERIOD_WITHOUT_DISCOUNT) {
+    return price;
+  }
+
+  if (days < PERIOD_WITH_SMALL_DISCOUNT) {
+    return (price -= SMALL_DISCOUNT);
+  }
+
+  return (price -= BIG_DISCOUNT);
```

Comment:
Similarly to the previous comment, use 'price - BIG_DISCOUNT' instead of modifying the 'price' variable with '-='.